analysis/src/GAutils.cpp

```
@@ -103,6 +118,15 @@ void ParseLogFile(std::string LogFile, int &nx, int &ny, int &nz, double &deltax
     }
     std::cout << "Number of layers in microstructure data: " << NumberOfLayers << std::endl;
     InputDataStream.close();
+
+    // If log file contained the X, Y, Z bounds of the domain and they're used in the analysis, convert them
```

Won't the parsing above fail if the bounds aren't included?

---

Yes - only ExaCA simulation results starting with this commit are compatible with this new analysis executable. I don't really see a reason why I'd need to run this new analysis script using any of the older datasets, though I should probably throw a specific error if I accidentally do try to run this with an old dataset